crowbar_framework/config/routes.rb

```diff
@@ -123,5 +123,9 @@
   get "crowbar/:barclamp", action: "versions", controller: "barclamp"
   post "crowbar/:barclamp/1.0/:action/:id", controller: "barclamp"
 
+  get "installer", controller: "installer", action: "index", as: "installer"
```

Comment:
I agree that how we write our routes is not optimal but should we start now and have two different styles? Or should we do this once after the merge?